### PR TITLE
Show Group properties (number of tabs and Group name)

### PR DESCRIPTION
**Pull request by akshgpt7**
What works:
- Show and stores the group name
- Show the number of tabs in the group
- Allow user to change group name

### CODE REVIEW

**Comment by daemon1024** [comment]
> * Put popup buttons on top?

I have another plan for this and it would be out of scope for this PR.

File: css/style.css [hunk 1]
```diff
@@ -107,12 +107,28 @@ button.bullet {
   padding: 0.5em;
 }
 
+
+
```
Comment (daemon1024) [inline css/style.css:111]: What's with all this padding xD 😂

Just add a comment instead, these new lines may cause conflicts later on.

**Comment by daemon1024** [comment]
Rest LGTM for now.

File: css/style.css (hunk 1 above)
Comment (akshgpt7) [reply css/style.css:111]: That's why it's WIP :p
Will remove them once it's final.

File: js/tab-manager.js [hunk 2]
```diff
@@ -64,20 +102,25 @@ function displayGroupList() {
...
 
     let deleteBtn = document.createElement("button");
     deleteBtn.className = "delete";
     deleteBtn.innerHTML = "Delete";
     deleteBtn.setAttribute("prop", prop);
-    header.appendChild(deleteBtn);
+    groupButtons.appendChild(deleteBtn);
 
     groupElement.appendChild(header);
+    header.appendChild(groupButtons);
```
Comment (daemon1024) [inline js/tab-manager.js:123]: 🎉

File: js/tab-manager.js [hunk 3]
```diff
@@ -103,19 +146,23 @@ document.getElementById("group-list").addEventListener("click", (e) => {
 
   if (e.target && e.target.matches("button.delete-tab")) {
     let prop = e.target.getAttribute("prop");
-    let i = e.target.getAttribute("i");
-    store[prop].splice(i, 1);
+    let i = e.target.getAttribute("index");
```
Comment (daemon1024) [inline js/tab-manager.js:149]: 😥

**Comment by akshgpt7** [comment]
@daemon1024 Everything works fine now, all that is left is styling the name and number of tabs properly, then it's ready to merge.

File: css/style.css [hunk 4]
```diff
@@ -114,25 +114,31 @@ button.add-tab {
...
+  display: flex;
+  flex-direction: row-reverse;
 }
 
 .group-name {
   background-color: transparent;
-  color: #bbe1fa;
+  color: #21e6c1;
   border: unset;
-  font-size: x-large;
```
Comment (akshgpt7) [inline css/style.css]: I think the group name looks too small right now. Looks a little odd with the number of tabs.
Maybe set this to `large`?

**Review by akshgpt7 — COMMENTED** [review]
LGTM!

**Pull request merged**